src/ksym.rs

```diff
@@ -52,6 +52,10 @@ impl KSymResolver {
             }
             let (addr, _symbol, func) = (tokens[0], tokens[1], tokens[2]);
             if let Ok(addr) = u64::from_str_radix(addr, 16) {
+                if addr == 0 {
+                    line.truncate(0);
```

Comment:
I feel like [`String::clear()`](https://doc.rust-lang.org/std/string/struct.String.html#method.clear) conveys intent more succinctly. Consider switching?

Reply:
Please consider adding a unit test triggering this case (without using `/proc/kallsyms`). Can be a follow up change.